_episodes/12-automatically-testing-software.md

```diff
@@ -1,7 +1,6 @@
 ---
 title: "Automatically Testing your Software"
-start: true
-teaching: 30
+steaching: 30
```

Comment:
Typo in episode metadata

Reply:
> With the changes to page titles, the title no longer appears on the main page: http://localhost:4000

Hmm, I thought I fixed that. Will have another look. -thx for spotting @jag1g13.

Reply:
Fixed via c6b51d7.